test/JDBC/input/test_windows_login-vu-prepare.sql

```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE ad_db;
+GO
+
+CREATE LOGIN [ad\Aduser] from windows;
+GO
+
+CREATE LOGIN [ad\Aduserdb] from windows with default_database=[ad_db];
+GO
+
+CREATE LOGIN [ad\Aduserlanguage] from windows with default_language=[German];
+GO
+
+CREATE LOGIN [ad\Aduserdblanguage] from windows with default_database=[ad_db], default_language=[German];
```

Comment:
We need to add a lot of tests here.
1. CREATE LOGIN with invalid login pattern for windows
2. duplicate login (created from TDS/created from PG, with password/with windows)
3. duplicate login with different letter-casing
4. DROP LOGIN
5. DROP LOGIN with login@domain format
6. DROP LOGIN for non-existent login
7. DROP login with different letter-casing.

All the valid test should be associated with a select from pg_auth_id and babelfish_authid_login_ext to verify the inputs.

Reply:
Done!